scripts/Init-Baking-Vars.ps1

```diff
@@ -13,10 +13,10 @@ Initialise the baking environment
 #>
 
 
-$script:SG = "bake-ami"
+$script:SG = "$script:VersionText"
 $script:externalip = $null
-$script:keypair = "RobG_id_rsa"
-$script:keypairfile = "$ENV:USERPROFILE\\.ssh\\id_rsa"
+$script:keypair = "Praveen"
+$script:keypairfile = "$ENV:USERPROFILE\\Downloads\\Praveen.pem"
```

Comment:
This change is too low level for a personalised setting.
Move `$script:keypair` & `$script:keypairfile` to `bake-ide-ps1` and set them from parameters passed into `bake-ide-ami.ps1`